packages/components/switch/__tests__/switch.test.tsx

```diff
@@ -111,8 +110,8 @@ describe('Switch.vue', () => {
   test('change event', async () => {
     const target = ref(1)
     const value = ref(true)
-    const handleChange = (val: boolean) => {
-      target.value = val
+    const handleChange = (val: string | number | boolean) => {
+      target.value = val as number
```

Comment:
I think passing a generic parameter to target ref is better than using type assertion.

```ts
const target = ref<string | number | boolean>(1)
```